CARDS-2424 / CARDS-2427: Configuration of export exclusions

sdumitriu opened the pull request:
This needs to be merged with #1660

To test:
- build with `mvn clean install -Pdocker`
- in `compose-cluster` run `python3 generate_compose_yaml.py --dev_docker_image  --composum --cards_project cards4heracles --s3_test_container --oak_filesystem`
- start with `docker-compose build && docker-compose up`
- at `http://localhost:9001/` create a new bucket called `uhn`
- fill in a Phone call follow-up form, including details about the providers visited
- open `http://localhost:8080/Subjects.s3push?dateLowerBound=2020-01-01&dateUpperBound=2024-01-01` to trigger an export
- at `http://localhost:9001/` check the pushed file, make sure all the answer except for the providers are included in it
- stop docker and cleanup with `docker-compose rm -vf && ./cleanup.sh`

sdumitriu committed:
CARDS-2424 / CARDS-2427: Configuration of export exclusions

sashaandjic reviewed (approved):
tested
-----
![image](https://github.com/data-team-uhn/cards/assets/23243244/e229e92e-9c21-4664-9f4c-2a809a98cd3a)
![image](https://github.com/data-team-uhn/cards/assets/23243244/67de5b16-9347-46ae-b545-870bfce6a8d0)

sdumitriu commented:
I don't quite like that we export an empty `Hospital visit:` and others, let me check if I can remove those.

marta- commented:
@sdumitriu we should also exclude:
* `section_admin/pc_call_initiated`
* `section_medication_changes/pc_conditions_summary`

sdumitriu committed:
CARDS-2424 / CARDS-2427: Configuration of export exclusions
Exclude the whole outer conditional section, not just the inner repeatable sections, since there is no other info inside them

sdumitriu committed:
CARDS-2424 / CARDS-2427: Configuration of export exclusions
Exclude two computed questions that are only useful in the UI

sashaandjic reviewed (approved):
tested
![image](https://github.com/data-team-uhn/cards/assets/23243244/000a5b21-2201-4059-a504-5b15c5a3c3ab)
![image](https://github.com/data-team-uhn/cards/assets/23243244/06087938-aec2-43af-b769-98fb6c4d1353)

![image](https://github.com/data-team-uhn/cards/assets/23243244/aca244fa-5a75-44bb-b77c-8bc4e7e862c2)

sashaandjic commented:
@sdumitriu Would you please also remove `pc_22_1section` as this section only contains an infobox that does not get exported.
![image](https://github.com/data-team-uhn/cards/assets/23243244/8068eb88-50e7-4ada-a340-876c91dcaff7)

sdumitriu committed:
CARDS-2424 / CARDS-2427: Configuration of export exclusions
One more empty section to exclude

sdumitriu committed:
CARDS-2424 / CARDS-2427: Configuration of export exclusions
The questionnaire name has changed
